### PR TITLE
[SSL] SSL/TLS DCV rewrite + other small tickets

### PR DESCRIPTION
**Pull request by kodster28**
Addresses:
- PCX-3590
- PCX-3569
- PCX-3516
- PCX-3522 (partially)

Also involved some pretty hefty rewrites to the DCV section of SSL/TLS.

### CODE REVIEW

File: content/ssl/edge-certificates/universal-ssl/disable-universal-ssl.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+pcx-content-type: how-to
+title: Disable Universal SSL certificates
+weight: 3
+---
+
+# Disable Universal SSL certificates
```
Comment (kodster28) [inline content/ssl/edge-certificates/universal-ssl/disable-universal-ssl.md:7]: Separated existing content into a different page.

File: content/_redirects [hunk 2]
```diff
@@ -254,7 +254,7 @@
 /logs/logpush/logpush-dashboard /logs/get-started/logpush-dashboard 302
 /logs/logpush/logpush-dashboard/ /logs/get-started/enable-destinations/ 302
 /logs/logpush/s3-compatible-endpoints /logs/get-started/enable-destinations/s3-compatible-endpoints 302
-/logs/logpush/splunk logs/get-started/enable-destinations/splunk 302
+/logs/logpush/splunk /logs/get-started/enable-destinations/splunk 302
```
Comment (marciocloudflare) [inline content/_redirects]:
```suggestion
/logs/logpush/splunk /logs/get-started/enable-destinations/splunk 301
```

Comment (marciocloudflare) [reply content/_redirects]: 301 is the permanent redirect, 302 is a temporary redirect. Feel free to ignore this if for some reason the redirect is temporary, though

File: content/ssl/_partials/_generic-view-validation-status.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+_build:
+  publishResources: false
+  render: never
+  list: never
+---
+
+Once you specify your chosen validation method, you can access the validation values by:
+
+- Going to **SSL/TLS** > **Edge Certificates** in the dashboard and clicking a certificate.
+- Getting certificate details [via the API](https://api.cloudflare.com/#certificate-packs-get-certificate-pack) and finding the `validation_method` and `validation_records`.
```
Comment (marciocloudflare) [inline content/ssl/_partials/_generic-view-validation-status.md]:
```suggestion
- Getting certificate details [via the API](https://api.cloudflare.com/#certificate-packs-get-certificate-pack), and finding the `validation_method` and `validation_records`.
```

File: content/ssl/_partials/_issue-certs-preamble.md [hunk 4]
```diff
@@ -7,4 +7,4 @@ _build:
 
 For each custom hostname certificate you request, Cloudflare issues two certificates that are bundled in chains that maximize browser compatibility (unless you [upload custom certificates](/ssl/ssl-for-saas/custom-certificates/uploading-certificates/)). The primary certificate uses a P-256 key, is SHA-2/ECDSA signed, and will be presented to browsers that support elliptic curve cryptography (ECC). The secondary or fallback certificate uses an RSA 2048-bit key, is SHA-2/RSA signed, and will be presented to browsers that do not support ECC.
 
-Once issued, certificates are valid for 1 year and renew automatically 30 days before expiration. Renewals require no action from you or your customer.
+Once issued, certificates are valid for 1 year. Renewals depend on your chosen method for [Domain Control Validation](/ssl/ssl-for-saas/common-tasks/certificate-validation-methods/).
```
Comment (marciocloudflare) [inline content/ssl/_partials/_issue-certs-preamble.md]:
```suggestion
Once issued, certificates are valid for one year. Renewals depend on your chosen method for [Domain Control Validation](/ssl/ssl-for-saas/common-tasks/certificate-validation-methods/).
```

File: content/ssl/_partials/_lets-encrypt-advanced-limitations.md [hunk 5]
```diff
@@ -0,0 +1 @@
+Selecting **Let’s Encrypt** as a CA limits a certificate to txt validation_method, 90 validity_days, omission of cloudflare_branding, and 2 host entries (one for the zone name and one for the subdomain wildcard of the zone name, such as `example.com`, `*.example.com`).
```
Comment (marciocloudflare) [inline content/ssl/_partials/_lets-encrypt-advanced-limitations.md]:
```suggestion
Selecting **Let’s Encrypt** as a CA limits a certificate to txt validation_method, 90 validity_days, omission of cloudflare_branding, and two host entries (one for the zone name and one for the subdomain wildcard of the zone name, such as `example.com`, `*.example.com`).
```

Comment (marciocloudflare) [reply content/ssl/_partials/_lets-encrypt-advanced-limitations.md]: I feel that `txt validation_method, 90 validity_days, omission of cloudflare_branding` should probably be between backticks but I'm not entirely sure

File: content/ssl/edge-certificates/universal-ssl/disable-universal-ssl.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+  - [Page rules](https://support.cloudflare.com/hc/articles/218411427) that redirect traffic to HTTPS
+  - HTTP to HTTPS redirects at your origin web server
+
+## Disable Universal SSL certificate
+
+To disable Universal SSL:
+
+1.  Make sure you have [uploaded a custom certificate](/ssl/edge-certificates/custom-certificates/) or purchased [Advanced Certificate Manager](/ssl/edge-certificates/advanced-certificate-manager/) to protect your domain.
+2.  Log in to the Cloudflare dashboard and select your account.
```
Comment (marciocloudflare) [inline content/ssl/edge-certificates/universal-ssl/disable-universal-ssl.md]:
```suggestion
2.  Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/login) and select your account.
```

**Review by marciocloudflare — APPROVED** [review]

File: content/_redirects (hunk 2 above)
Comment (pedrosousa) [reply content/_redirects]: I already fixed a couple of missing slashes (including this one) in PR #3958 — sorry, didn't see this PR coming. Not sure why your change here is not causing a conflict, though... 
I'd merge `production` to this PR's source branch and check for duplicate lines in the `_redirects` file, just to be sure.

File: content/ssl/_partials/_lets-encrypt-advanced-limitations.md (hunk 5 above)
Comment (kodster28) [reply content/ssl/_partials/_lets-encrypt-advanced-limitations.md]: Addressed with another commit

**Pull request merged**